remix/app/routes/_app.newsletter.update.$id.tsx

```diff
@@ -58,19 +78,31 @@ export default function Index() {
   }, [result]);
 
   return (
-    <div>
-      <Form method="put">
-        <div>
-          <label htmlFor="name">Name</label>
-          <input type="text" name="name" placeholder="name" />
-        </div>
-        <br />
-        <div>
-          <label htmlFor="content">Content</label>
-          <textarea name="content" placeholder="content" />
+    <section className="flex flex-col gap-4">
+      <header className="text-xl font-bold">
+        <h1>Update your Newsletter </h1>
+      </header>
+      <ValidatedForm
+        validator={validator}
+        defaultValues={newsletter}
+        method="put"
+      >
+        <div className="flex flex-col gap-6">
+          <FormInput
+            label={'Name'}
```

Comment:
Can we just remove all brackets like this 👀 :

```typescript
  <FormInput
    label="Content"
    name="content"
    placeholder="content"
    format="long"
  />
```

Reply:
Yep, I just realize that for the `normal string` props that are not a variable, it is better to use it without `{}` which helps use to virtualize which one is just a `string` or `variable` that passes to the component.

Good suggestion @tednguyendev  💪🏽